<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnsharpMask</class>
 <widget class="QWidget" name="UnsharpMask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>43</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>43</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Maska</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pAnuluj">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- Cofnij</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pOk">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pPrzycisk">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Zastosuj</string>
   </property>
  </widget>
  <widget class="QSlider" name="sigmaSlider">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>30</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sigma</string>
   </property>
  </widget>
  <widget class="QSlider" name="alfaSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>26</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Alfa</string>
   </property>
  </widget>
  <widget class="QLabel" name="al">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="sig">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
